src/apps/expenses/components/CreateExpenseForm.js

```diff
@@ -504,15 +504,15 @@ class CreateExpenseForm extends React.Component {
   }
 
   render() {
-    const { LoggedInUser, collective } = this.props;
+    const { LoggedInUser } = this.props;
 
     if (!LoggedInUser) {
       return (
         <div className="CreateExpenseForm">
-          <P textAlign="center" mt={4} fontSize="LeadParagraph" lineHeight="LeadParagraph">
-            <FormattedMessage id="expenses.create.login" defaultMessage="Sign up or login to submit an expense." />
+          <P textAlign="center" mt={4} fontSize="LeadParagraph" lineHeight="LeadParagraph" mb={4}>
+            <FormattedMessage id="expenses.create.login" defaultMessage="Sign up or login to submit an expense" />
           </P>
-          <SignInForm next={`/${collective.slug}/expenses/new`} />
+          <SignInOrJoinFree />
```

Comment:
Would be great to have some margins here

![image](https://user-images.githubusercontent.com/1556356/59904694-7c6fea00-9404-11e9-8e1e-ba9305420add.png)

Reply:
Done